syntax = "proto3";

package Pb;

message ClientHead
{
   int32     sync_id = 1;
   int64     req_time = 2;
}